### PR TITLE
Use .Net Core task to install .Net 6

### PR DESCRIPTION
**Pull request by Francisco-Gamino**
### Issue describing the changes in this PR

Use .Net Core task to install `.Net 6`.

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: azure-pipelines-e2e-integration-tests.yml [hunk 1]
```diff
@@ -18,13 +18,13 @@ pool:
 
 steps:
 - pwsh: |
```
Comment (ejizba) [inline azure-pipelines-e2e-integration-tests.yml]: Have you tried using the built-in dotnet install task? I think that would be way easier...

Here's where we use it in node.js:
https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/azure-pipelines/test.yml#L91

Here's the docs:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops

Comment (Francisco-Gamino) [reply azure-pipelines-e2e-integration-tests.yml]: Great point @ejizba . I will give this a try.

**Review by michaelpeng36 — APPROVED** [review]
LGTM

**Review by ejizba — APPROVED** [review]

**Comment by Francisco-Gamino** [comment]
This PR is replaced by https://github.com/Azure/azure-functions-powershell-worker/pull/750.

**Pull request closed without merging**